flashinfer_trace/tests/references/test_gdn_prefill_qk16_v32_d128.py

```
@@ -0,0 +1,431 @@
+        (2, 16),  # Two sequences, medium
+        (4, 32),  # Multiple sequences, longer
+    ]
+
+    passed = 0
+    total = len(test_configs) + 1  # +1 for no initial state test
+
+    for num_seqs, max_seq_len in test_configs:
+        try:
+            if test_correctness(num_seqs, max_seq_len):
+                passed += 1
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)